pkgs/applications/misc/countdown/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+buildGoPackage rec {
+  name = "countdown-${version}";
+  version = "1.0.0";
+
+  goPackagePath = "github.com/antonmedv/countdown";
+
+  src = fetchFromGitHub {
+    owner = "antonmedv";
+    repo = "countdown";
+    rev = "v${version}";
+    sha256 = "0pdaw1krr0bsl4amhwx03v2b02iznvwvqn7af5zp4fkzjaj14cdw";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/antonmedv/countdown;
+    description = "Terminal countdown timer";
+    maintainers = with maintainers; [ ];
```

Comment:
Are you sure you don't want to add yourself as a maintainer of this package?

Reply:
I've added myself as a maintainer for the package!